### PR TITLE
Remove isodd

### PR DESCRIPTION
**Pull request by tonetheman**
I removed is-odd from your dependencies. It looked like you used an npm package called is-odd.

It felt like you might not need a package to do that small of a test. No worries if you choose not to take the PR.

After my change I checked that all tests still passed. I did not add any new tests since the change looked like it would be tested during the negate portion of your existing tests.

### CODE REVIEW

File: lib/parsers.js [hunk 1]
```diff
@@ -94,7 +93,7 @@ module.exports = function(nanomatch, options) {
       if (!m) return;
       var val = m[0];
 
-      var isNegated = isOdd(val.length);
+      var isNegated = Math.abs(val.length%2)==1;
```
Comment (rehevkor5) [inline lib/parsers.js]: This change is great, but I doubt you need to worry about strings with negative length :)

Comment (tonetheman) [reply lib/parsers.js]: Yeah just being careful. I can take out the Math.abs if you want :)

Comment (rehevkor5) [reply lib/parsers.js]: It seems like being careful is how we got to `isOdd` in the first place.

File: lib/parsers.js [hunk 2]
```diff
@@ -94,7 +93,7 @@ module.exports = function(nanomatch, options) {
       if (!m) return;
       var val = m[0];
 
-      var isNegated = isOdd(val.length);
+      var isNegated = ((val.length%2)==1);
```
Comment (doowb) [inline lib/parsers.js]: Nitpicks:
 - remove the outer parentheses
 - put spaces before and after the `%` and the `==`
 - use strict equal `===`

Basically: `var isNegated = (val.length % 2) === 1;`

**Comment by mc10** [comment]
This looks identical to the patch in #7.

**Comment by doowb** [comment]
@tonetheman thanks for the PR!

/cc @jonschlinkert @phated @es128 if there are no objects, I think this can be published as a patch bump.

**Comment by jonschlinkert** [comment]
ftr, this is the last PR that will be accepted for is-odd or is-even. we don't have time for trolls.

**Comment by jonschlinkert** [comment]
ftr, this is the last PR that will be accepted for is-odd or is-even. we don't have time for trolls.

**Pull request merged**